tests/tr1/prefix.lst

```diff
@@ -3,4 +3,4 @@
 
 RUNALL_INCLUDE ..\universal_prefix.lst
 RUNALL_CROSSLIST
-PM_CL="/D_ENFORCE_FACET_SPECIALIZATIONS=1 /D_CRT_SECURE_NO_WARNINGS"
+PM_CL="/FIforce_include.hpp /D_ENFORCE_FACET_SPECIALIZATIONS=1 /D_CRT_SECURE_NO_WARNINGS"

```

Comment:
We should note that the importer will need to verify that `std/include` is made available to `tr1` when running in the internal test harness.

Reply:
Excellent catch, I forgot that Microsoft-internal `src/qa/distrib.yaml` would need to be changed. Thanks!